cloud-inquisitor/serverless/tag_auditor/main.go

```diff
@@ -4,55 +4,55 @@ import (
 	"context"
 
 	"github.com/RiotGames/cloud-inquisitor/cloud-inquisitor"
-	"github.com/aws/aws-lambda-go/lambda"
-
-	log "github.com/sirupsen/logrus"
+	"github.com/RiotGames/cloud-inquisitor/cloud-inquisitor/instrumentation/newrelic"
+	"github.com/RiotGames/cloud-inquisitor/cloud-inquisitor/settings"
 )
 
 func handlerRequest(ctx context.Context, resource cloudinquisitor.PassableResource) (cloudinquisitor.PassableResource, error) {
-	log.Printf("resource: %#v\n", resource)
-	parsedResource, err := resource.GetResource()
+
+	parsedResource, err := resource.GetResource(ctx, map[string]interface{}{
```

Comment:
Same as the logger in the previous review...
if ctx won't change and will be used in lots of different places, do you think it is ok to make it global?